andela/trime
# Fix profile page visualisations

### What does this PR do?
Fixes profile page visualisations not showing.

### Description of Task to be completed?
Fix profile page visualisations not showing.
Fix profile page visualisations using the data from one project for all projects.

### How should this be manually tested?
Open the profile page. You should see pie charts for each project you've tracked time on.

### What are the relevant pivotal tracker stories?
[Visualisations are not displayed on the profile page](https://www.pivotaltracker.com/story/show/128309417)
